CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+TODO: add contributing documentation
```

Comment:
We should adapt the one from `tektoncd/catalog` for this repo I think.

Reply:
Yeah, I will use it as a template. But there are some changes we need to make; for example, using git based versioning and the new automated testing setup.

Will add the details in later PRs if you don't mind 😁